src/CSF.SRDashboard.Client/Services/MtoaArtifactService.cs

```diff
@@ -0,0 +1,183 @@
+﻿using CSF.SRDashboard.Client.DTO;
+using CSF.SRDashboard.Client.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CSF.Web.Client.Utilities;
+using System.Net.Http;
+
+namespace CSF.SRDashboard.Client.Services
+{
+    public class MtoaArtifactService : IMtoaArtifactService
+    {
+        private readonly IConfiguration configuration;
+        private readonly IRestClient restClient;
+        private readonly ILogger<MtoaArtifactService> logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MtoaServices"/> class.
+        /// </summary>
+        /// <param name="configuration">Application configuration service.</param>
+        /// <param name="restClient">REST client service.</param>
+        /// <param name="logger">Application logger service.</param>
+        public MtoaArtifactService(IConfiguration configuration, IRestClient restClient, ILogger<MtoaArtifactService> logger)
+        {
+            this.configuration = configuration;
+            this.restClient = restClient;
+            this.logger = logger;
+        }
+
+        public int GetNumberOfRequestForSeafarers()
+        {
+            int numberOfRequest = -1;
+
+            bool excludeMetadata = true;
+            bool excludeServiceRequests = true;
+            int page = 1;
+            int pageSize = 1;
+
+            string serviceId = this.configuration.GetSection("MtoaServiceSettings")["ServiceId"];
+            string pathTemplate = this.configuration.GetSection("MtoaServiceSettings")["GetServiceRequestsPath"];
+            string path = string.Format(pathTemplate, serviceId, page, pageSize, excludeMetadata, excludeServiceRequests);
+
+            var serviceRequestInfo = restClient.GetAsync<ServiceRequestCollection>(ServiceLocatorDomain.Mtoa, path).GetAwaiter().GetResult();
+            numberOfRequest = serviceRequestInfo.Count;
+
+            return numberOfRequest;
+        }
+
+        public List<ServiceRequest> GetAllRequestsForSeafarers(int numberOfRequest)
```

Comment:
I'm not sure we ever want to get all requests I would change the name of the parameter. We probably want to move this to MtoaService class.
```suggestion
        public List<ServiceRequest> GetAllRequestsForSeafarers(int pageSize)
```

Reply:
Getting all the request was the idea that we came up with the team I believe. Tibo suggested this way of loading data.